applications/nrf_desktop/src/events/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/selector_event.c)
 
 target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/hid_event.c)
 
+target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/passkey_event.c)
```

Comment:
Please put the above sources into the same "target_sources" invocation to cut down on boilerplate.

target_sources(app PRIVATE
  ${CMAKE_CURRENT_SOURCE_DIR}/passkey_event.c
  ${CMAKE_CURRENT_SOURCE_DIR}/hid_event.c
  [etc]  
)

Reply:
The unconditional sources below should also be included.

Reply:
I will add this one in separate commit.